core/LSU.cpp

```diff
@@ -257,7 +263,7 @@ namespace olympia
         }
 
         // Access TLB, and check TLB hit or miss
-        bool TLB_HIT = MMULookup_(mem_access_info_ptr);
+        bool TLB_HIT = mmu_->Lookup(mem_access_info_ptr);
```

Comment:
Two things:
1. While we haven't really established coding style guidelines, this variable name should not be in all caps.  Request `tlb_hit` to match the other code
2. This variable can be `const` : `const bool tlb_hit =`.  This is helpful is avoiding accidental assignments when you intended to compare.